tgui/.gitignore

```diff
@@ -13,8 +13,8 @@ package-lock.json
 
 ## Build artifacts
 /public/.tmp/**/*
-##/public/**/*
-##!/public/*.html
+/public/**/*
+!/public/*.html
```

Comment:
Not persuaded this is usefull, but it's like that in TG

Reply:
thats gitignore stuff

Reply:
Definitly 100% needed